README.md

```diff
@@ -111,3 +111,9 @@ Run all tests (`nvim` must be in the `PATH`):
 Run only the `TestMessageDeserialization` test:
 
     dotnet test --filter TestMessageDeserialization test/NvimClient.Test/NvimClient.Test.csproj
+
+Publishing GitHub Package
```

Comment:
It's a nit but how's this title? `Publishing` could also work, or `Release`, or . . . ?

Reply:
just `Release`, as in pynvim. We want all release steps to be captured in one place. "Publishing" is over-specific

Reply:
Should I rename the workflow from `publish.yml` -> `release.yml` as well, so that's consistent?

Reply:
> Should I rename the workflow from `publish.yml` -> `release.yml` as well, so that's consistent?

up to you. The CI jobs may be necessarily more granular than the README section :)

it's a reversible a decision